src/Wellcome.Dds/Wellcome.Dds.Repositories/Wellcome.Dds.Repositories.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.3" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="ShellProgressBar" Version="5.1.0" />
```

Comment:
Feels odd to take a dependency on a package for console apps in the Repositories project, the web projects will have indirect dep on this. Would it be better to move `DumpUtils` into the Catalogue tool and have WorkflowProcessor reference that?

Reply:
Yeah, I'll do that